lib/codebreaker/hmi.rb

```diff
@@ -0,0 +1,89 @@
+module Codebreaker
+  class Hmi
+    def initialize
+      @game_stop = false
+      @hmi_commands = %w[help hint save replay exit show_score erase_score]
+      @message = GameAppMessages.new
+    end
+
+    def game_init
+      puts @message.show_message(:game_intro)
+      puts @message.show_message(:login)
+      @codebreaker_name = gets.chomp
+      @game = Game.new(@codebreaker_name)
+      puts @message.show_message(:game_start, name: @codebreaker_name)
+      launch_game
```

Comment:
After game will end you need to propose for gamer show all results, restart game or exit from game. Add this feature, please.